test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Aggregation/AggregationTests.cs

```diff
@@ -684,7 +684,7 @@ public async Task ComputeAfterGroupByWorks()
                 string.Format(
                     AggregationTestBaseUrl + "?$apply=groupby((Name), aggregate(Order/Price with sum as TotalAmount))"
                     + "/compute(TotalAmount mul 2 as DoubleAmount, length(Name) as NameLen)"
-                    + "&$orderby=TotalAmount",
+                    + "&$orderby=TotalAmount&$filter=Name ne null",
```

Comment:
It should be $apply=...../filter(Name null)/compute(...)

Spec says that $apply should be evaluated first and only then $filter. 
As a result, for case with $filter, we will generate LINQ that looks like:
```C#
.Select(x =>new {
        Name = x.Name,
        NameLen=x.Name.Length
})
.Where(x => x.Name != null)
```
I guess .NET or EF optimizes real execution by swapping .Select and .Where. However, that behavior may change.

Reply:
I changed the query, thanks.